.github/workflows/ci-report.yml

```diff
@@ -0,0 +1,55 @@
+name: "CI Report"
+
+on:
+  workflow_run:
+    workflows:
+      - "CI"
+    types:
+      - completed
+
+jobs:
+  test-results:
+    name: "Publish Test Results"
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
```

Comment:
When would it be skipped when the event is completed?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run

Could be an `if` on it (which there's none, yet), but I think cancelled is also modelled as completed based on the 3 states. Got it from the action author's recommendation:
https://github.com/EnricoMi/publish-unit-test-result-action/blob/v2.16.1/README.md#support-fork-repositories-and-dependabot-branches

Reply:
Well, tested that manual cancellation is not a "skipped" conclusion, so thanks for highlighting.

Reply:
For reference if you're curious (manually cancelled run)
![image](https://github.com/TWiStErRob/net.twisterrob.ghlint/assets/2906988/8d128165-5318-4c32-8bdf-eff4b1d92b45)